src/honeysql_postgres/helpers.cljc

```diff
@@ -1,6 +1,6 @@
 (ns honeysql-postgres.helpers
   (:refer-clojure :exclude [partition-by])
-  (:require [honeysql-postgres.format :refer :all]
```

Comment:
We need to do this because:

* clj-kondo does not like `:refer :all`
* ClojureScript does not support it, but we do test against it (as we should)